src/screens/Login/Login.js

```diff
@@ -1,21 +1,31 @@
 import React, {Component} from 'react';
-import {Text, View, ScrollView, Image, TextInput, LayoutAnimation} from 'react-native';
+import {Text, View, ScrollView, Image, TextInput, LayoutAnimation, Vibration} from 'react-native';
 import CustomModal from 'src/components/CustomModal';
 import {AnimatedButton} from 'src/components/CustomButton';
 import styles from './styles';
 import {authenticate} from 'src/api/auth';
 import {flow} from 'src/helpers/lodash';
+import {putIntoStorage, getFromStorage} from 'src/helpers/asyncStorage';
 
 class Login extends Component {
-  state = { username: '', password: '', loading: false };
+  state = { username: '', password: '', loading: false, isStorageChecked: false };
+
+  componentDidMount() {
+    getFromStorage('isLogined')
```

Comment:
It's not a good idea do it in cdm lifecycle method, because if you logout you will not be able to login using another login and password i.e.

I think better store some session info ( token or username and password i.e. ) and check it before sending creds to the server.